### DISCUSSION TITLE
Scrolling through long beatmap's leaderboard causes fps drops and freezes

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19057] [Q&A]
https://user-images.githubusercontent.com/86934170/178095030-fadb01d2-3060-4379-880a-528bd726e708.mp4

https://osu.ppy.sh/beatmapsets/750284#taiko/1904970

[runtime.log](https://github.com/ppy/osu/files/9076018/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9076019/updater.log)
[network.log](https://github.com/ppy/osu/files/9076016/network.log)
[performance.log](https://github.com/ppy/osu/files/9076017/performance.log)

**Comment by nagi-desuuu** [comment]
From what I'm seeing, those other player's abysmally high scores might be the cause of the problem. That, and I think repeatedly loading and rendering cover images might be the assistant to the crash.
Did you try switching the scoring system to standardized?

**Comment by frenzibyte** [comment]
Logs don't show any errors, contrary to what the video shows.

**Comment by smoogipoo** [comment]
I believe this is going to be diffcalc related.

**Reply by Loreos7** [reply]
I guess it really is

https://user-images.githubusercontent.com/86934170/179226677-b61e2c67-6540-4994-906f-ddf9e1515731.mp4
